src/main/java/com/global/api/gateways/RealexConnector.java

```diff
@@ -476,6 +480,25 @@ public Transaction manageTransaction(ManagementBuilder builder) throws ApiExcept
         et.subElement(request, "channel", channel);
         et.subElement(request, "orderid", orderId);
         et.subElement(request, "pasref", builder.getTransactionId());
+        
+        if (builder.getAuthorizationCode() != null) {
+            et.subElement(request, "authcode").text(builder.getAuthorizationCode());
```

Comment:
can be shortened to:
et.subElement(request, "authcode", builder.getAuthorizationCode()));